test/lib/Elastica/Test/Facet/DateHistogramTest.php

```diff
@@ -28,22 +28,23 @@ public function testTest()
 
         $mapping = new Mapping($type, array(
                 'name' => array('type' => 'string', 'store' => 'no'),
-                'dtmPosted' => array('type' => 'date', 'store' => 'no', 'format' => 'yyyy-MM-dd HH:mm:ss'),
+                'dtmPosted' => array('type' => 'long', 'store' => 'no'),
```

Comment:
Yeah. I'll keep the original test and add a new one instead to check that `factor` works.